src/components/FormNavigation/FormNavigation.tsx

```diff
@@ -24,29 +27,49 @@ const FormNavigation: React.FC<FormNavigationProps> = ({
   onFormSave,
   previousPage,
   onDestroySession,
-}) => (
-  <ButtonGroup className="openforms-form-navigation" direction="column">
-    {submitButton}
-
-    {previousPage && <PreviousLink to={previousPage} onClick={onNavigatePrevPage} position="end" />}
-
-    {onFormSave && (
-      <LinkButton
-        name="save"
-        onClick={onFormSave}
-        className="openforms-form-navigation__save-button"
-      >
-        <Icon>
-          <FAIcon icon="" className="fa-fw" />
-        </Icon>
-        <Literal name="saveText" />
-      </LinkButton>
-    )}
-
-    {!hideAbortButton && (
-      <AbortButton isAuthenticated={isAuthenticated} onDestroySession={onDestroySession} />
-    )}
-  </ButtonGroup>
-);
+}) => {
+  const {backToTopText, backToTopRef} = useContext(ConfigContext);
+  const backToTopNode = backToTopRef ? document.getElementById(backToTopRef) : null;
```

Comment:
doing DOM queries during React component rendering doesn't fee like something that should be allowed. Either way it's quite wasteful if you could (if you absolute need the DOM Node reference instead of just the ID value) instead resolve the element in `sdk.tsx` and put that in the configuration context instead.